microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -6,79 +6,91 @@ ms.author: msedgedevrel
+The Evergreen runtime updates automatically on the client, so that the latest features and security patches are available to your WebView2 app.  The Evergreen runtime also requires less storage space on the disk than the Fixed Version runtime.  Fixed Version runtime distribution is only recommended for apps that have strict compatibility requirements.
 
-<!-- ====================================================================== -->
-## Run compatibility tests regularly when using the Evergreen Runtime
+For more information about the benefits of the Evergreen Runtime, see [Details about the Evergreen Runtime distribution mode](../concepts/distribution.md#details-about-the-evergreen-runtime-distribution-mode) in _Distribute your app and the WebView2 Runtime_.  
 
-When using the Evergreen WebView2 Runtime, the runtime updates automatically, so you must regularly run compatibility tests.  To ensure that your WebView2 app will continue to work as expected, test your web content in the WebView2 control against the Microsoft Edge preview channels (Beta, Dev, or Canary).  The  preview channels are also called _Insider channels_.
+To help ensure that your WebView2 app works well with the Evergreen WebView2 Runtime, use the recommended practices in the subsections below:
+* [Check to make sure the WebView2 Runtime is installed](#check-to-make-sure-the-webview2-runtime-is-installed).
+* [Handle updates of the WebView2 Runtime](#handle-updates-of-the-webview2-runtime).
+* [Do forward-compatibility testing using Edge preview channels](#do-forward-compatibility-testing-using-edge-preview-channels).
+* [Use feature detection for newer APIs](#use-feature-detection-for-newer-apis).
 
-The above guidance is similar to the guidance for web developers.  To test your app for forward-compatibility, see [Prerelease testing using preview channels](../how-to/prerelease-testing.md) and [Self-host by deploying preview channels](../how-to/self-hosting.md).
 
-To download the preview channels of Microsoft Edge, go to [Become a Microsoft Edge Insider](https://www.microsoft.com/edge/download/insider).
+<!-- ------------------------------ -->
+#### Check to make sure the WebView2 Runtime is installed
 
+When using the Evergreen WebView2 Runtime, before your app uses the WebView2 control, programmatically check to make sure that the WebView2 Runtime is installed on the client.
+<!-- todo: check specifically for the "Evergreen" runtime?   which to say?
+check to make sure that the WebView2 Runtime is installed
+check to make sure that the Evergreen WebView2 Runtime is installed
+-->
 
-<!-- ====================================================================== -->
-## Test whether newer APIs are supported by the installed WebView2 Runtime
+See [Detect if a suitable WebView2 Runtime is already installed](../concepts/distribution.md#detect-if-a-suitable-webview2-runtime-is-already-installed) in _Distribute your app and the WebView2 Runtime_.
```

Comment:
the word "suitable" seems wrong & should be removed from the heading.  The test is binary yes/no is the wv2 Runtime installed or not.  It's not a test for which version of wv2 Runtime is installed & whether that version is suitable?

rendered article section: 
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?branch=pr-en-us-3195&tabs=dotnetcsharp#detect-if-a-suitable-webview2-runtime-is-already-installed

Reply:
@mikehoffms, the destination article is actually about testing if WebView2 is installed + what version of WebView2 is installed. I think that's why the word "suitable" was used to encompass both aspects.

I do agree that "suitable" feels not quite right.

Reply:
removed suitable from heading & link text.